Proposals/XXXX-scaling-codeless-community.md

```diff
@@ -0,0 +1,71 @@
+# Scaling A Codeless Open Source Swift Community with Bas Broek
+
+* Proposal: [PR#22](https://github.com/SwiftCommunityPodcast/podcast/pull/22)
+
+* Authors: [Garric Nahapetian](https://github.com//garricn)
```

Comment:
So it seems like these URLs work regardless, but both this one and Chris' (in the review manager line) have two forward slashes; I think that should just be one?